logparser/devices/markdownformatdevice.py

```diff
@@ -128,6 +129,8 @@ def write_configurations(self, state):
             self.write_statistics_bandwidth(state)
         if 'statistics_packet' in state and not state['no_stats']:
             self.write_statistics_packets(state)
+        if 'threads' in state and not state['no_stats']:
```

Comment:
If the argument is "no_", should be directly:
```python
if 'threads' in state and state['no_stats']:
```

Please, could you change this in ```_initialize_state```? Thank you